rspace/README.md

```diff
@@ -6,25 +6,25 @@ The **R**Chain Tuple **Space**
 
-Traditionally, a key-value store provides two main functions for interacting with a given store: *put*, which persists a piece of data at given key, and *get*, which retrieves the data stored at a particular key.
+Traditionally, a key-value store provides two main functions for interacting with a given store: *put*, which persists a piece of data at a given key, and *get*, which retrieves the data stored at a particular key.
 
 `rspace` departs from this model and provides a novel approach to storing and retrieving data.
 
 Here are some key differences:
 
-* Rather than associating a piece of data with a particular key, in `rspace`, a piece of data is associated with a *channel*.  Roughly speaking, in `rspace`, a *channel* can be any Scala type that can be serialized to and deserialized from `Array[Byte]`.
+* Rather than associating a piece of data with a particular key, in `rspace`, a piece of data is associated with a *channel*.  Roughly speaking, in `rspace`, a *channel* can be any Scala type that can be serialized to and deserialized from an `Array[Byte]`.
 
-* In addition to storing data, `rspace` can also store *continuations* which represent actions to carry out once the data is retrieved.  Roughly speaking, in `rspace`, a *continuation* can be any Scala type that can be serialized to and deserialized from`Array[Byte]`.
+* In addition to storing data, `rspace` can also store *continuations*, which represent actions to carry out once the data is retrieved.  Roughly speaking, in `rspace`, a *continuation* can be any Scala type that can be serialized to and deserialized from an`Array[Byte]`.
 
-* A continuation is associated with a list of channels and a list of *patterns*.  Like channels and continuations, patterns can be any Scala type that can be serialized to and deserialized from `Array[Byte]`.  Users of `rspace` provide their own custom match function for matching patterns against data.
+* A continuation is associated with a list of channels and a list of *patterns*.  Like channels and continuations, patterns can be any Scala type that can be serialized to and deserialized from an `Array[Byte]`.  Users of `rspace` provide their own custom match function for matching patterns against data.
 
 * The two main functions for interacting with a given store are:
 
   * *consume*, which searches the store for data matching a given list of patterns at a given list channels; and
 
   * *produce*, which, given a piece data at a given channel, searches the store for matching continuation, using existing data in the store to help satisfy the match.
 
-For more information and detailed instructions on using `rspace`, see the tutorial here.
+For more information and detailed instructions on using `rspace`, see the tutorial [here]().
 
```

Comment:
Are you planning on adding the actual link later?

Reply:
Yes